app/javascript/controllers/teams_tab_change_controller.js

```diff
@@ -1,34 +1,23 @@
 import { Controller } from "@hotwired/stimulus";
 
 export default class extends Controller {
-  static targets = ["teamContent", "allUsersSection", "showAllLink"];
+  static targets = ["teamContent"];
```

Comment:
we probably should name both user tab and team tab as separate or just name it as a generic name like "tabContent".

Reply:
teamContent has changed to tabContent